Makefile

```diff
@@ -120,6 +130,14 @@ test: unit_test dm_unit_test engine_unit_test
 
 build: cdc dm engine
 
+check-makefiles: ## Check the makefiles format.
+check-makefiles: format-makefiles
+	@git diff --exit-code -- $(MAKE_FILES) || (echo "Please format Makefiles by running 'make format-makefiles'" && false)
+
+format-makefiles: ## Format all Makefiles.
+format-makefiles: $(MAKE_FILES)
+	$(SED) -i -e 's/^\(\t*\)  /\1\t/g' -e 's/^\(\t*\) /\1/' -- $?
```

Comment:
No need to address. It works fine if the indent of tab is 2 or 4 spaces, but in 4 spaces scenario we need to run `make format-makefiles` multiple times to replace all spaces.

Reply:
Good catch! 
Thanks for your review!  💚 💙 💜 💛 ❤️